metadata/src/test/java/org/apache/kafka/image/FeaturesImageTest.java

```diff
@@ -163,7 +163,7 @@ public void testEmpty() {
         assertFalse(new FeaturesImage(Collections.singletonMap("foo", (short) 1),
             FeaturesImage.EMPTY.metadataVersion()).isEmpty());
         assertFalse(new FeaturesImage(FeaturesImage.EMPTY.finalizedVersions(),
-            MetadataVersion.IBP_3_3_IV0).isEmpty());
+            MetadataVersion.IBP_3_4_IV0).isEmpty());
```

Comment:
Since this can be any MV, could we just use MetadataVersion.MINIMUM_VERSION?

Reply:
That causes the test to fail. This is related to your question of whether `FeatuesImage.isEmpty` should be changed.

Reply:
I updated `FeaturesImage.isEmpty` and also this test. Resolving.